indico/modules/users/export_schemas.py

```diff
@@ -0,0 +1,432 @@
+        return any(link.is_speaker for link in contrib.person_links if link.person.user == user)
+
+    def _get_minutes(self, contrib):
+        if note := contrib.note:
+            return note.current_revision.html
+
+
+class AbstractFileExportSchema(mm.SQLAlchemyAutoSchema):
+    class Meta:
+        model = AbstractFile
+        fields = ('id', 'filename', 'md5', 'url', '_archive_path')
+
+    url = fields.Function(lambda f: url_for('abstracts.download_attachment', f, management=False, _external=True))
+    _archive_path = fields.Function(lambda file: build_storage_path(file))
+
+
+class AbstractExportSchema(mm.SQLAlchemyAutoSchema):
+    class Meta:
+        model = Abstract
+        fields = ('id', 'contribution_id', 'event_id', 'title', 'description', 'files',
+                  'submission_comment', 'submitted_dt', 'state', 'url', 'event_url')
+
+    description = fields.String(attribute='_description')
+    state = fields.Enum(AbstractState)
+    url = fields.Function(lambda abstract: url_for('abstracts.display_abstract', abstract, management=False,
+                                                   _external=True))
+    event_url = fields.Function(lambda abstract: url_for('events.display', abstract.event, _external=True))
+    files = fields.List(fields.Nested(AbstractFileExportSchema))
+
+
+class PaperFileExportSchema(mm.SQLAlchemyAutoSchema):
+    class Meta:
+        model = PaperFile
+        fields = ('id', 'filename', 'md5', 'url', '_archive_path')
+
+    url = fields.Function(lambda f: url_for('papers.download_file', f.paper.contribution, f, _external=True))
+    path = fields.Function(lambda file: build_storage_path(file))
+
+
+class PaperExportSchema(mm.SQLAlchemyAutoSchema):
+    class Meta:
+        model = Contribution
+        fields = ('id', 'title', 'description', 'submitted_dt', 'url', 'event_url', 'files')
+
+    submitted_dt = fields.Method('_get_submitted_dt')
+    state = fields.Enum(PaperRevisionState)
+    url = fields.Function(lambda paper: url_for('papers.paper_timeline', paper, _external=True))
+    event_url = fields.Function(lambda paper: url_for('events.display', paper.event, _external=True))
+    files = fields.List(fields.Nested(PaperFileExportSchema))
+
+    def _get_submitted_dt(self, paper):
+        return fields.DateTime().serialize('submitted_dt', paper.revisions[0])
+
+
+class PersonalDataExportSchema(mm.SQLAlchemyAutoSchema):
+    class Meta:
+        model = User
+        fields = ('first_name', 'last_name', 'full_name', 'title', 'email', 'secondary_emails',
+                  'affiliation', 'phone', 'avatar_url', 'picture_metadata', 'picture_source',
+                  'favorite_events', 'favorite_categories')
+
+    title = NoneValueEnumField(UserTitle, none_value=UserTitle.none, attribute='_title')
+    secondary_emails = fields.Function(lambda user: list(user.secondary_emails))
+    affiliation = fields.Nested(AffiliationSchema, attribute='affiliation_link')
```

Comment:
You need to include both `affiliation` and `affiliation_link` - the latter is often None (in case of the usual free-text affiliations).

Also, exclude the `meta` field the affiliation link. That's internal and not relevant for users.